doc/api_assets/api.js

```diff
@@ -136,6 +136,29 @@
     updateHashes();
   }
 
+  function setupFlavorSelectors() {
```

Comment:
```suggestion
  function setupFlavorToggles() {
```

Reply:
This comment applies to all current suggestions:

If you want to change every mention of "select" to "toggle", then we also need to change all the CSS files and also all the existing functions in `api.js` and possibly more. There are also test cases that might be affected.

I think this renaming thing should go in a separate issue so we can land this feature. No need to spend time risking introducing other issues on something that is not really a problem imho.

Reply:
I think I've done all the preparation work in https://github.com/nodejs/node/pull/49536, do you think I've missed something? We can't land this PR as is because it's now inconsistent with `main`, but I think all you need to do is to merge my suggestions and this can land as soon as we have a green CI.

Reply:
Ok, so you've committed the renaming directly onto main already?

The "setupCopyButton" function in `doc/api_assets/api.js` also uses the name "selector" everywhere, is that included? Not sure what else.

Reply:
Pushed merge and rename.